build/commands/lib/util.js

```diff
@@ -513,13 +513,13 @@ const util = {
 
     console.log('generate Widevine sig files...')
 
-    util.run('python', [sig_generator, '--input_file=' + path.join(src_dir, 'brave.exe'),
+    util.run('python3', [sig_generator, '--input_file=' + path.join(src_dir, 'brave.exe'),
```

Comment:
Hmm, I think using `python3` here doesn't give much benefit for widevine sig file generation.
IIRC, the issue was depot tools's python doesn't support `cryptography` module.
If it's supported, I think we can do this via gn target as @bridiver commented above.

Reply:
Also, newer cryptography versions don't even support Python 2 anymore.